yt_dlp/extractor/youtube.py

```diff
@@ -90,6 +93,7 @@
             },
         },
         'INNERTUBE_CONTEXT_CLIENT_NAME': 1,
+        'REQUIRE_PO_TOKEN': True,
     },
     'web_embedded': {
```

Comment:
`WEB_EMBEDDED_PLAYER` doesn't require a PO token?

Reply:
Not currently last time I tested